cmd/main.go

```diff
@@ -240,11 +241,15 @@ func main() {
 
 	h := handlers.NewHandlerInstance(hc, meshsyncCh, log, brokerConn)
 
+	b := broadcast.NewBroadcaster(100)
```

Comment:
100 is the channel buffer length, correct? What's the significance of 100? What factors lead us to this size?

Reply:
Nothing particular significance.